README.md

```diff
@@ -52,19 +51,19 @@ restful-service
 	        └── order_mgt_service_test.bal
 ```
 
-- Create the above directories in your local machine and also create empty `.bal` files.
+- Create the above directories on your local machine. Additionally, create an empty `.bal` file too.
 
-- Then open the terminal and navigate to `restful-service/guide` and run Ballerina project initializing toolkit.
+- Open the terminal, navigate to `restful-service/guide`, and run the Ballerina project initializing toolkit.
 ```bash
    $ ballerina init
 ```
 
 ### Developing the RESTful web service
 
-- We can get started with a Ballerina service; 'order_mgt_service', which is the RESTful service that serves the order management request. order_mgt_service can have multiple resources and each resource is dedicated for a specific order management functionality.
+- Let's start implementing the Ballerina service; `order_mgt_service`, which is the RESTful service that serves the order management requests. The `order_mgt_service` has multiple resources and each resource is dedicated to a specific order management functionality.
 
-- You can add the content to your Ballerina service as shown below. In that code segment you can find the implementation of the service and resource skeletons of 'order_mgt_service'. 
-For each order management operation, there is a dedicated resource and inside each resource we can implement the order management operation logic. 
+- Add the content given below to your Ballerina service. The code segment includes the implementation of the service and the resource skeleton of the `order_mgt_service`. 
+There is a dedicated resource for each order management operation. You can implement the order management operation logic inside each resource. 
 
 ##### Skeleton code for order_mgt_service.bal
```

Comment:
@kasun04: 
Suggestion: Do you think we should remove the skeleton and only keep the code for the order_mgt_service.bal file? Because the user will get the idea from the order_mgt_service.bal file straight away.